### ISSUE TITLE
Training configuration on KINS dataset

### ISSUE THREAD

**Issue by trqminh** [YutingXiao/Amodal-Segmentation-Based-on-Visible-Region-Segmentation-and-Shape-Prior#13]
Hi @YutingXiao,
Thank you for your consideration. 

I have a question about reproducing results. Indeed, I am reproducing Mask RCNN performance on the KINS dataset. I follow[ this config file ](https://github.com/YutingXiao/Amodal-Segmentation-Based-on-Visible-Region-Segmentation-and-Shape-Prior/blob/main/configs/KINS-AmodalSegmentation/mask_rcnn_R_101_FPN_1x_amodal.yaml)to train, but my results are lower than those reported in Table 1 of the paper (e.g. AP of maskrcnn: 27.91 of mine compare to 30.01 of yours). Did you use the same config as I am using here?

Below I add the screenshot of my results and yours so you can have a clearer look. Thank you!

![image](https://user-images.githubusercontent.com/30286786/174648813-bc52682e-db1b-48bd-bb8e-bb1033263260.png)
![image](https://user-images.githubusercontent.com/30286786/174648716-6c2c558e-279b-431d-81d2-bf2a996cb85e.png)

**Comment by trqminh** [comment]
My bad. I forgot to change the pre-trained weight to R101 in the config file.
